src/Symfony/Component/Validator/Tests/Constraints/BackedEnumValueTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Validator\Constraints\BackedEnumValue;
+use Symfony\Component\Validator\Mapping\ClassMetadata;
+use Symfony\Component\Validator\Mapping\Loader\AttributeLoader;
+
+/**
+ * @author Aurélien Pillevesse <aurelienpillevesse@hotmail.fr>
+ */
+class BackedEnumValueTest extends TestCase
+{
+    public function testAttributes()
```

Comment:
we should also add tests covering the cases that throw exceptions

Reply:
@AurelienPillevesse this one is not resolved also

Reply:
There is here : https://github.com/symfony/symfony/blob/fb6f09418e1d85937387dd2b459ac11db22f7969/src/Symfony/Component/Validator/Tests/Constraints/BackedEnumValueValidatorTest.php#L31

Which case do you want to add ?